src/store/cockpit/cockpitApi.ts

```diff
@@ -652,6 +652,23 @@ export const cockpitApi = contentSourcesApi.injectEndpoints({
               });
             });
 
+            // TODO: maybe give a warning that we need admin access?
+            await cockpit.spawn(['systemctl', 'stop', 'osbuild-*'], {
+              superuser: 'require',
+            });
+
+            await cockpit.spawn(
+              [
+                'systemctl',
+                'start',
+                'osbuild-composer.socket',
+                'osbuild-composer',
```

Comment:
I'm not even sure if we need to restart composer, just the worker(s) should be fine.

`osbuild-worker@*.service`

Reply:
Okay cool, let me try that out

Reply:
ah ok, ye let's just restart all of them. lgtm thanks!